### PR TITLE
MERN_MongoDB: 1.1 What is MongoDB?(V)

### LINKED ISSUE #2252: MERN_MongoDB: 1.1 What is MongoDB?(V)
## What is MongoDB?
Provide the .md file containing the drive link for MP4 video explaining "**Introduction to MongoDB in MERN stack**".

## Project directory
`Branch MERN>MERN/Topic/Backend/MongoDB/`

## File Name
`1.1_What_is_MongoDB?(V).md`

### Domain

MERN

### Type of Contribution

Video

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by pankaj-Makhijani**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #2252 

<hr>

## Type of change 📝

- [x] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)

<hr>

## Domain of Contribution 📊

- [x] MERN

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: MERN/Topic/Backend/MongoDB/1.1 What is MongoDB(V).md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# What is Node.js?
```
Comment (saloni-15) [inline MERN/Topic/Backend/MongoDB/1.1 What is MongoDB(V).md:1]: Looks good to me ... just ~Node.js~ MongoDB :)

**Review by saloni-15 — CHANGES_REQUESTED** [review]

File: MERN/Topic/Backend/MongoDB/1.1 What is MongoDB(V).md (hunk 1 above)
Comment (pankaj-Makhijani) [reply MERN/Topic/Backend/MongoDB/1.1 What is MongoDB(V).md]: Made requested changes.

**Review by saloni-15 — APPROVED** [review]

**Review by Naman-1234 — APPROVED** [review]

**Comment by pankaj-Makhijani** [comment]
Hello @saloni-15, you approved the changes but my PR is not merged.

**Comment by saloni-15** [comment]
> Hello @saloni-15, you approved the changes but my PR is not merged.

Hey, actually I'm mentor and write access is given to supervisors so don't worry supervisor will checkout and merge this.

**Pull request merged**